week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -38,7 +58,19 @@ Write a function that:
 */
 
 function formatPercentage(arr) {
-  
+  let myArray = arr;
+  let newArray = [];
+  let formatArray = [];
+  let finalArray = [];
+  for (let i = 0; i < myArray.length; i++) {
+    newArray[i] = Math.round(arr[i] * 100) / 100;
+    if (newArray[i] > 100) {
+      newArray[i] = 100;
+    }
+    formatArray.push(newArray[i]);
```

Comment:
Do we need the `formatArray` here? Does it differ from the `newArray`? Try printing them both out and comparing to see.